Mailer/app/api/v1/Handlers/Mailers/MailTypesListingHandler.php

```diff
@@ -18,19 +20,36 @@ public function __construct(
         $this->listsRepository = $listsRepository;
     }
 
-    public function handle($params)
+    public function handle($payload)
     {
+        $request = file_get_contents("php://input");
+        $payload = $this->validateInput($request, 'mail-types.schema.json');
```

Comment:
We shouldn't be redeclaring function call parameter which hasn't even been used. Based on the parent class, it should remain `handle($params)`

Reply:
I used as example original file SendEmailHandler where there is $payload used as parameter in handle.

Reply:
There's `$params` in master :thinking:

Reply:
I'm talking about this one: https://github.com/remp2020/remp/blob/e97ef9053cb9e28efc94d0f440178107263876a0/Mailer/app/api/v1/Handlers/Mailers/SendEmailHandler.php#L46

Reply:
Please fix that one too (rename to `$params` in `SendEmailHandler`). Funny that IDE plugin doesn't think the one in `SendEmailHandler` is problematic, but in this class it is (because of the try/catch).

Reply:
Done